source/examples/cuml-snowflake-nb/notebook.ipynb

```diff
@@ -0,0 +1,555 @@
+{
```

Comment:
<p>We aren't using the IPython magic method here. We should update the language.</p><p> </p><p>Also is this definitely correct? I know in <code>cudf.pandas</code> you have to restart the kernel, importing again doesn't work. Does the kernel need to be restarted here too?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/590/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>